config/jobs/kubernetes-csi/external-attacher/external-attacher-config.yaml

```diff
@@ -279,9 +243,9 @@ presubmits:
         # unrelated to the PR. Testing against the latest Kubernetes is covered
         # by periodic jobs (see https://k8s-testgrid.appspot.com/sig-storage-csi#Summary).
         - name: CSI_PROW_KUBERNETES_VERSION
-          value: "1.14.0"
+          value: "1.15"
```

Comment:
I would prefer to keep `CSI_PROW_KUBERNETES_VERSION` with all three places. I know that the last digit might get patched, but is that always the case?

Reply:
With the latest prow.sh, it doesn't get patched if:
* You specify a version outside of 1.13-1.15
* You specify "release-1.x"

Reply:
done